src/js/add_input_icon.js

```diff
@@ -123,7 +123,7 @@ async function addRelayIconToInput(emailInput) {
   const emailInputOriginalParentEl = emailInput.parentElement;
 
   // create new wrapping element;
-  const emailInputWrapper = createElementWithClassList("div", "fx-relay-email-input-wrapper");
+  const emailInputWrapper = createElementWithClassList("span", "fx-relay-email-input-wrapper");
```

Comment:
Would it be possible to use a custom element (via `customElements.define…`?) and avoid inherited styles? I say that bc this change will absolutely fix Issue #1114, but it may open us up to entirely new website issues who have styles applied to `spans` in their email field/login areas.

Reply:
Yeah that's definitely something to consider, depending on whether that interferes with host styling on the wrapped input. Additionally, I think we'd want to consider using shadow DOM for at least the Relay icon, to avoid all the `!important`s in our stylesheet.

Reply:
Ahhh. I love that it. I'll file that for a seperate enhancement. Great idea!